.openshift-ci/compatibility_test.py

```
@@ -18,48 +21,101 @@
+    """Shallow-clone the helm-charts repo and return (clone_dir, file:// URL).
+
+    Uses GITHUB_TOKEN for authenticated access when available.
+    Returns (None, fallback_url) on failure.
+    """
+    clone_dir = tempfile.mkdtemp(prefix="helm-charts-clone-")
+    repo_url = HELM_CHARTS_GIT_REPO
+    token = os.environ.get("GITHUB_TOKEN", "")
+    if token:
+        repo_url = f"https://x-access-token:{token}@github.com/stackrox/helm-charts.git"
+    try:
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)